tests/main/helpers/test_ods.py

```diff
@@ -1,11 +1,10 @@
 import mock
-import unittest
 import functools
 
 import app.main.helpers.ods as ods
 
 from hypothesis import strategies as st
-from hypothesis import given, example, assume
+from hypothesis import given, example
 
```

Comment:
I'm pretty _partial_ to removing this pattern whenever I find it.
LOOOOOL

But not in this PR cause it doesn't cause a flake8 error on its own. No matter how much I wish it did.